blocks/library/gallery/index.js

```diff
@@ -167,13 +167,49 @@ registerBlockType( 'core/gallery', {
 					const img = <img src={ image.url } alt={ image.alt } data-id={ image.id } />;
 
 					return (
-						<figure key={ image.id || image.url } className="blocks-gallery-image">
-							{ href ? <a href={ href }>{ img }</a> : img }
-						</figure>
+						<li key={ image.id || image.url } className="blocks-gallery-item">
+							<figure>
+								{ href ? <a href={ href }>{ img }</a> : img }
+							</figure>
+						</li>
 					);
 				} ) }
-			</div>
+			</ul>
 		);
 	},
 
+	deprecated: [
```

Comment:
In the deprecated section, you can just paste the old "attributes" and "save" functions before the change.

Reply:
All of them, or just the changed attributes?